### PR TITLE
contributor and hirearchy fixes

### PR DESCRIPTION
**Pull request by mfranz1**
This PR is for the story: https://app.clubhouse.io/clarkcan/story/4173/last-round-of-contributors-updates

I removed a line of code that prints the 'author' of the learning object in the learning-object component and removed the title cases for parent/child hierarchies. I tested using 'ng serve --configuration=production' and the changes seemed to have worked.

image
image

### CODE REVIEW

File: src/app/cube/details/details.component.html [hunk 1]
```diff
@@ -90,10 +90,6 @@ <h2 class='section-title'>Reviews</h2>
             </cube-details-action-panel>
           </section>
           <section class='details__content-container__right-side-container'>
-            <h3 class='section-title'>Authors</h3>
```
Comment (pzalep1) [inline src/app/cube/details/details.component.html:93]: This needs to stay put. We need them on the details page but not on the cards.

File: src/app/cube/shared/learning-object/learning-object.component.html [hunk 2]
```diff
@@ -51,7 +51,6 @@
       </div>
       <div class="details">
         <span class="author">
-          {{ learningObject.author.name | titlecase }} |
           {{ learningObject.author.organization.trim() | titlecase
```
Comment (pzalep1) [inline src/app/cube/shared/learning-object/learning-object.component.html:54]: We can remove this whole thing.

**Review by pzalep1 — CHANGES_REQUESTED** [review]
For the contributors we should show the first contributors name and org.

File: src/app/cube/details/details.component.html (hunk 1 above)
Comment (mfranz1) [reply src/app/cube/details/details.component.html:93]: Ah I see, my bad

**Review by jmulholland4014 — APPROVED** [review]
After Paige's comments looks good!

**Review by EVogel1999 — APPROVED** [review]
Same as John's comment.  After Paige's comments are addressed it looks good!  Thank you!

**Review by Cwagne17 — APPROVED** [review]

File: src/app/cube/shared/learning-object/learning-object.component.html [hunk 3]
```diff
@@ -51,9 +51,8 @@
       </div>
       <div class="details">
         <span class="author">
-          {{ learningObject.author.name | titlecase }} |
-          {{ learningObject.author.organization.trim() | titlecase
-          }}{{
+          {{learningObject.contributors[0]?.name | titlecase}}
+          {{
             learningObject.contributors?.length
               ? ', and ' + learningObject.contributors.length + ' more'
```
Comment (pzalep1) [inline src/app/cube/shared/learning-object/learning-object.component.html:57]: Subtract one from the length so it is accurate. You're displaying the first one.

**Review by pzalep1 — CHANGES_REQUESTED** [review]
Can we get a screen shot too?

**Review by pzalep1 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Review by EVogel1999 — APPROVED** [review]
Good job on this!  Know it was a pain to figure out.

**Pull request merged**